packages/reakit/src/Toolbar/Toolbar.tsx

```diff
@@ -48,4 +55,6 @@ Toolbar.defaultProps = {
   gutter: 8
 };
 
-export default as("div")(Toolbar);
+const ToolbarComponent = as("div")(Toolbar);
+
+export default ToolbarComponent;
```

Comment:
Let's keep it as `export default as("div")(Toolbar);`.

Reply:
yes, didn't clean that yet